yarn-project/ethereum/package.json

```diff
@@ -25,6 +25,7 @@
   ],
   "dependencies": {
     "@aztec/foundation": "workspace:^",
+    "@aztec/l1-artifacts": "workspace:^",
```

Comment:
This seems like an anti-pattern. Pretty much everything in yarn-project will now depend on l1-artifacts when most of it shouldn't need to should it?

Reply:
Perhaps look to extract `ethereum` from so far up the dependency tree if we want to do this.

Reply:
Good point. I checked and negatively affected by this are these packages:
```
Boxes
Pxe
Sequencer-client
Circuit-types
aztec.js
Aztec-node
Aztec-faucet
```

I will rework this.